unittests/fork_test_utilities.cpp

```diff
@@ -9,10 +9,10 @@ public_key_type  get_public_key( name keyname, string role ){
 }
 
 void push_blocks( tester& from, tester& to, uint32_t block_num_limit ) {
-   while( to.control->fork_db_pending_head_block_num()
-            < std::min( from.control->fork_db_pending_head_block_num(), block_num_limit ) )
+   while( to.control->fork_db_head_block_num()
+            < std::min( from.control->fork_db_head_block_num(), block_num_limit ) )
    {
-      auto fb = from.control->fetch_block_by_number( to.control->fork_db_pending_head_block_num()+1 );
+      auto fb = from.control->fetch_block_by_number( to.control->fork_db_head_block_num()+1 );
```

Comment:
If this is used in non IRREVERSIBLE mode, it will not do the same thing. Is it on purpose?

Reply:
All tests pass ;). The point of this method is to copy blocks from one test chain to another. It uses `fetch_block_by_number` to do that, so it could only copy what that could find, so the actual functionality has not changed here.